nsxt/resource_nsxt_transport_node.go

```diff
@@ -494,8 +494,8 @@ func getEdgeNodeSchema() *schema.Schema {
 	return getNodeSchema(s, false)
 }
 
-func getHostNodeSchema() *schema.Schema {
-	s := map[string]*schema.Schema{
+func getHostNodeSchemaAddlElements() map[string]*schema.Schema {
```

Comment:
Sorry for being annoying about the naming - I'm guessing `addl` means `additional`? May I suggest `getSharedHostNodeSchemaAttrs` or something similar?